### PR TITLE
[ops] [throughput] Eliminate launcher no-candidate starvation streak

### LINKED ISSUE #691: [ops] [throughput] Eliminate launcher no-candidate starvation streak
## Why / Context
Generated from live event-lane telemetry and ranked by the opportunity synthesizer. Value score: 0.892; source type: launcher.starvation. If we improve opportunity generation quality, launcher no-candidate streaks should approach zero.

## Dependency Metadata
- Value score: 0.892

## Scope
**In scope**
- Investigate the evidence chain and isolate the smallest bounded fix that addresses the signal.
- Implement the remediation in the likely modules listed below.
- Add deterministic verification that demonstrates signal improvement after the change.

**Out of scope**
- Broad refactors or architectural rewrites unrelated to this signal.
- Unbounded scope growth across unrelated components.

Constraints:
- Keep changes small/mergeable.
- Deterministic tests only.
- Avoid touching unrelated generated artifacts unless required.

## Acceptance Criteria (required)
- [ ] Root cause for the opportunity signal is identified and remediated with bounded blast radius.
- [ ] Deterministic tests or scripted checks are added/updated and fail before the fix but pass after.
- [ ] PR description includes before/after evidence tied to the originating signal.

## Implementation Notes / Pointers
**Likely files / modules:**
- `opportunity:opp_17b60321e9`

**Related tests:**

**Edge cases to handle:**
- Signal may be intermittent; include at least one regression path for recurrence.
- Avoid introducing false positives in existing duplicate guards or cooldown logic.
- launcher.starvation (evt_ec9c41530716)
- launcher.starvation (evt_58007b190d71)
- launcher.starvation (evt_bc7f6c71cbbf)

## Commands (local)
- `make lint`
- `make test`

## PR Requirements
- PR title should match the issue.
- PR body must include: `Fixes #<issue-number>`
- CI must be green; if Agent Artifacts Freshness fails, run `uv run agent-regenerate`, then commit `var/agents/...` updates.

## Codex-Ready Checklist (for the issue creator)
- [x] Clear acceptance criteria
- [x] At least one file pointer
- [x] Commands included
- [x] No ambiguous "do the right thing" language

### PR DESCRIPTION
**Pull request by Solders-Girdles**
## Summary
- add a launcher starvation guard to monitoring builtins
- expose the guard through the monitoring guard package exports
- add dedicated unit coverage for starvation detection and reset behavior

Closes #691

### CODE REVIEW

File: src/gpt_trader/monitoring/guards/builtins.py [hunk 1]
```diff
@@ -184,10 +184,47 @@ def _evaluate(self, context: dict[str, Any]) -> tuple[bool, str]:
...
+        streak_raw = context.get("no_candidate_streak")
+        if streak_raw is None:
+            return False, ""
+
+        try:
+            streak_value = int(streak_raw)
+        except (TypeError, ValueError):
+            return False, ""
+
+        threshold_override = context.get("threshold_override") or context.get("threshold")
```
[the remark this answers is not in the document]
Comment (Solders-Girdles) [reply src/gpt_trader/monitoring/guards/builtins.py]: Fixed in 34ba09e5. `LauncherStarvationGuard` now only falls back to `threshold` when `threshold_override is None`, so an explicit override of `0` is preserved and honored. Added regression coverage in `test_zero_threshold_override_is_honored`.
